nav2_smoother/src/nav2_smoother.cpp

```diff
@@ -0,0 +1,298 @@
+// Copyright (c) 2019 RoboTech Vision
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <chrono>
+#include <limits>
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "nav2_core/exceptions.hpp"
+#include "nav2_smoother/nav2_smoother.hpp"
+#include "nav2_util/geometry_utils.hpp"
+#include "nav2_util/node_utils.hpp"
+#include "nav_2d_utils/conversions.hpp"
+#include "nav_2d_utils/tf_help.hpp"
+#include "tf2_ros/create_timer_ros.h"
+
+using namespace std::chrono_literals;
+
+namespace nav2_smoother {
+
+SmootherServer::SmootherServer()
+    : LifecycleNode("smoother_server", "", true),
```

Comment:
Set this to false, we no longer support use of multiple nodes per server. Please see other examples like nav2_planner for how to run with a single node and executors.

Reply:
@gezp any advise you can give him on this would be helpful